src/ppom-controller.test.ts

```diff
@@ -192,7 +192,32 @@ describe('PPOMController', () => {
       }).rejects.toThrow('User has securityAlertsEnabled set to false');
     });
 
-    it('should throw error if the user is not on ethereum mainnet', async () => {
+    it('should NOT throw error if the user is on any supported chain id', async () => {
+      buildFetchSpy();
+      ppomController = buildPPOMController({
+        chainId: '0x38',
+        state: {
```

Comment:
@jpuri My thought is that I can just pass only chainId here and it will pass, but it didn't. I need to also pass the state.

Reply:
Could you find reason why absence of state break the test here ?

Reply:
`no files found for the chainId I passed`